defaults.ini

```diff
@@ -10,6 +10,7 @@ db_password=
 db_port=
 db_prefix=
 logger_destination=file:data/logs/default.log
+; logger_level values: EMERGENCY, ALERT, CRITICAL, ERROR, WARNING, NOTICE, INFO, DEBUG, NONE
```

Comment:
I am not sure we want to duplicate the docs in the config. Instead I would update the link at the top to point to https://selfoss.aditu.de/#configuration_params, add [config validation](https://github.com/SSilence/selfoss/issues/1061) and [streamline the docs](https://github.com/SSilence/selfoss/issues/979).